.pre-commit-config.yaml

```diff
@@ -3,7 +3,9 @@ repos:
     rev: v1.46.0
     hooks:
       - id: terraform_fmt
+      - id: terraform_validate
       - id: terraform_docs
+      - id: terraform_tflint
```

Comment:
```suggestion
```

Reply:
Submodules are generated from the templates and Terraform validation will fail since templates are separated into pieces.

Reply:
It is safer to turn off such checks until the module is updated (it was created when Terraform 0.10 was released).